### ISSUE TITLE
Refactor Game.fellow_players to include all players?

### ISSUE THREAD

**Issue by PeterBorah** [PeterBorah/vigor#17]
Riot's Game API is designed such that it lists information for the player you searched for on the game object itself, and then leaves that player out of fellowPlayers. Should we hide this fact, and just embed the searched-for player in the list of players?

**Comment by imogenkinsman** [comment]
Makes sense to me - I don't think a game should necessarily be associated with a player, so maybe there should be a game#players instead that shows all players that were in the game.
